packages/appcd-util/src/util.js

```diff
@@ -427,6 +427,74 @@ export function mutex(name, callback) {
 		}));
 }
 
+/**
+ * Tries to resolve the operating system name and version.
+ *
+ * @returns {Object}
+ */
+export function osInfo() {
```

Comment:
I think we do something similar maybe in the system-info plugin. Should we maybe move that to use this function at some point?

Reply:
That's the idea. I'll update the system-info plugin to this new `osInfo()` in appcd-util once we publish it. :)